app/js/pages/Escrow/components/CardEscrow.jsx

```diff
@@ -1,13 +1,87 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Card, CardBody, Button } from 'reactstrap';
+import {FontAwesomeIcon} from "@fortawesome/react-fontawesome";
+import {faCircleNotch, faCheck} from "@fortawesome/free-solid-svg-icons";
+
+import Reputation from '../../../components/Reputation';
+import RoundedIcon from "../../../ui/RoundedIcon";
+
+import "./CardEscrow.scss";
+
+import one from "../../../../images/escrow/01.png";
+import two from "../../../../images/escrow/02.png";
+import three from "../../../../images/escrow/03.png";
+import four from "../../../../images/escrow/04.png";
+import five from "../../../../images/escrow/05.png";
+
+const Done = () => (
+  <React.Fragment>
+    <RoundedIcon icon={faCheck} bgColor="green"/>
+    <h2 className="mt-4">Done.</h2>
+    <h2 className="mt-4">Rate your trading experience with this user.</h2>
+    <Reputation reputation={{upCount: 1, downCount: 1}} size="l"/>
+  </React.Fragment>
+);
+
+const Releasing = () => (
+  <React.Fragment>
+    <span className="number text-white p-3 rounded-circle">
+      <img src={five} />
+    </span>
+    <h2 className="mt-4">Waiting for the confirmations from the miners</h2>
+    <FontAwesomeIcon icon={faCircleNotch} size="5x" spin/>
+  </React.Fragment>
+);
+
+const Funded = () => (
+  <React.Fragment>
+    <span className="number text-white p-3 rounded-circle">
+      <img src={four} />
+    </span>
+    <h2 className="mt-4">Funds are in the escrow. Release them when you will get the payment.</h2>
+    <Button color="primary" className="btn-lg mt-3" onClick={() => {}}>Release funds</Button>
+  </React.Fragment>
+);
+
+const Funding = () => (
+  <React.Fragment>
+    <span className="number text-white p-3 rounded-circle">
+      <img src={three} />
+    </span>
+    <h2 className="mt-4">Waiting for the confirmations from the miners</h2>
+    <FontAwesomeIcon icon={faCircleNotch} size="5x" spin/>
+  </React.Fragment>
+);
+
+const PreFund = () => (
+  <React.Fragment>
+    <span className="number text-white p-3 rounded-circle">
+      <img src={two} />
+    </span>
+    <h2 className="mt-4">You are about to send</h2>
+    <h2 className="text-success">2.0 ETH</h2>
+    <h2>+ our fee</h2>
+    <h2 className="text-success">1 SNT</h2>
```

Comment:
That's a lot of `h2`s lol. Maybe they should be `p`s with the `h2` class? Because those are not titles?

Reply:
I :heart: `h2`

Updated